### ISSUE TITLE
Unify documents format

### ISSUE THREAD

**Issue by leandro-lucarella-sociomantic** [sociomantic-tsunami/neptune#8]
For now we have 2 documents in rst and 1 in md, we should stick to just one format to keep things simpler.

**Comment by mathias-baumann-sociomantic** [comment]
It was argued here https://github.com/sociomantic-tsunami/sociomantic/pull/2#issuecomment-323861820 that the dominant format should be .md

**Comment by leandro-lucarella-sociomantic** [comment]
Yes, we are forced to use MD in some places already (GitHub releases and comments for example), so they only way to use only one format is to stick to MD.
